PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java

```diff
@@ -53,19 +52,19 @@ public RawMaterial(long invoiceID, long supplierID, String noteString) {
 	 * @param noteString
 	 * @param dateTime
 	 */
-	public RawMaterial(long invoiceID, long supplierID, String noteString, String dateTime) {
+	public InvoiceDetails(long invoiceID, long supplierID, String noteString, String dateTime) {
 		this.invoiceID = invoiceID;
 		this.supplierID = supplierID;
 		this.noteString = noteString;
 		this.dateTime = LocalDate.parse(dateTime);
 	}
 
 	public long getRawMaterialID() {
```

Comment:
and setters and getters

Reply:
No need. it is already done on my side. ignore it please